ansible/roles/beetbox-drupal/templates/uploadprogress.ini.j2

```diff
@@ -0,0 +1,2 @@
+; activate and configurate for php uploadprogress module
```

Comment:
Given the non-templatiness of this template, could this become a generic extension template with two vars?

`{{ extension }} & {{ extension_extras }}`